test/inductor/test_fp8.py

```diff
@@ -94,7 +94,7 @@ def f(x):
 
     @parametrize("dtype", (torch.float16, torch.bfloat16))
     def test_eager_fallback(self, dtype: torch.dtype, device: torch.device):
-        if device == "cuda" and not PLATFORM_SUPPORTS_FP8:
+        if "cuda" in device and not PLATFORM_SUPPORTS_FP8:
```

Comment:
This test module seems inconsistent.  Some params are typed as `device: str` and others are `device: torch.device`.  In the latter case, torch.device has the `type` attribute.  torch.device is not iterable so `"cuda" in device` I expect to fail.

```suggestion
        if device.type == "cuda" and not PLATFORM_SUPPORTS_FP8:
```